lib/auth/domain/auth_service.dart

```diff
@@ -13,4 +13,8 @@ class AuthService {
   Future<bool> inLoggedIn() async {
     return await _authRepository.inLoggedIn();
```

Comment:
remove `await`
`return` by default will use `await` even if you didn't write `await`